src/views/CVERecord/PublishedRecord.vue

```diff
@@ -353,34 +137,119 @@ export default ({
         references: [],
         tags: [],
       },
-      originalRecordData: this.$store.state.recordData || {},
+      originalRecordData: usecveRecordStore().recordData || {},
+      cveServicesBaseUrl: import.meta.env.VITE_CVE_SERVICES_BASE_URL,
+      usecveRecordStore: usecveRecordStore(),
+      cpe: {},
+      cnaContainer: {},
+      cveProgramContainer: {},
+      adpContainers: {},
+      orgIdAndLongNameMap: {},
     };
   },
   methods: {
+    toggleAccordian() {
```

Comment:
I fixed instances of this in other pages on my `jd-3000-accordion` branch, but it should be spelled accordion instead of accordian

Reply:
Fixed typo for entire app.